datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/AbstractKafkaConnector.java

```diff
@@ -553,4 +570,12 @@ public AbstractKafkaBasedConnectorTask getConnectorTask() {
       return _task;
     }
   }
+
+  @Override
+  public List<BrooklinMetricInfo> getMetricInfos() {
```

Comment:
@ahmedahamid and @vishwajith-s I just noticed that callers of this pass the "connectorName" when fetching metrics from the connectorTask. Should we also only return metrics for a given "connectorName" here?

Reply:
@vishwajith-s had a chat with @ahmedahamid about this offline. We not need to pass the "connectorName" here, as the only metric available is the one we added as part of this PR and it is keyed based on the "connectorName". For the tasks, it looks like the method needed to be static for some reason, or it was an oversight, which is why the "connectorName" is passed.

Reply:
@somandal with the recent changes, this may not be required.

Reply:
connector name is being passed as part of metrics prefix